libr/core/cbin.c

```diff
@@ -1300,6 +1300,7 @@ static char *get_reloc_name(RCore *r, RBinReloc *reloc, ut64 addr) {
 		// TODO(eddyb) implement constant relocs.
 	}
 	free (demangled_name);
+	r_name_filter(reloc_name, -1);
```

Comment:
missing space

Reply:
also this namefilter should be happening before printing, its fine if the reloc name contains special chars here